lib/core/widgets/side_navbar.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+class SideNavBar extends StatelessWidget {
+  const SideNavBar({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    const navBarItems = ['Home', 'Reservations', 'Guests'];
+    final theme = Theme.of(context).colorScheme;
+
+    return Padding(
+      padding: const EdgeInsets.all(16),
+      child: Drawer(
+        shape: const RoundedRectangleBorder(
+            borderRadius: BorderRadius.all(Radius.circular(20))),
+        child: Container(
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(20),
+              gradient: LinearGradient(
+                begin: const Alignment(0, -1.3),
+                end: const Alignment(0, 6),
+                colors: <Color>[
+                  const Color(0xff271039).withOpacity(0.7),
+                  const Color(0xff404040).withOpacity(0.3),
+                ],
+                tileMode: TileMode.mirror,
+              ),
+            ),
+            child: Padding(
+              padding: const EdgeInsets.only(top: 20, bottom: 20),
+              child: ListView.separated(
+                  itemBuilder: (context, index) => TextButton(
+                      onPressed: () {
+                        // navigate to page
+                      },
+                      style: ButtonStyle(
+                          overlayColor:
+                              MaterialStateProperty.all(Colors.transparent)),
+                      child: Padding(
+                        padding: const EdgeInsets.all(10.0),
+                        child: Text(
+                          navBarItems[index],
+                          style: TextStyle(
+                              color: theme.onSurface,
+                              fontSize: 22,
```

Comment:
In the default size upon opening the application the text for Reservations seems to break to a new line. Make sure to either decrease the font size or the padding around the text 

![image](https://user-images.githubusercontent.com/34609899/195913310-127e4bd3-f95c-4195-ae6f-b3970ef11a43.png)

Reply:
after upgrading to the latest flutter version, this seems to be a non problem. lgtm